docs/UserGuide.md

```diff
@@ -686,8 +697,8 @@ Example(s):
 
 Marks an existing order in the order list as paid and/or delivered. 
 
-<div markdown="block" class="alert alert-info">
-:information_source: **Warning:** `marko` is irreversible. This means that you cannot unmark an order that is marked as 
+<div markdown="span" class="alert alert-info">
```

Comment:
also the class here should be `"alert alert-warning"` so that the info box will be yellow instead of the normal blue!